python/goodData_cff.py

```diff
@@ -10,7 +10,7 @@
                                   L1GtReadoutRecordTag = cms.InputTag('gtDigis')
                                   )
 
-primaryVertexFilter = cms.EDFilter('GoodVertexFilter',
+primaryVertex = cms.EDFilter('GoodVertexFilter',
```

Comment:
Hi Guys,

sorry for the delay in answering, but as you know we have tight dead lines.
Indeedmy proposal is to wait a couple of weeks before taking decisions
about this PR, just to have enough time for deep checks.

b.t.w...
quickly looking at the code I don’t aspect big problems since there are new
files (for miniAOD) and it seams that Kurt is not changing to much in
common files.

possible issues:
1. Kurt, maybe wrongly, changed the name of a filter that I changed to fix
a problem I recently had compiling in 7_4_12:
https://github.com/cms-analysis/SUSYBSMAnalysis
-Zprime2muAnalysis/pull/14/files#diff-eeac8088a38b3a15a6cd08c0080bf702L13
(question from benjmin in previous mail)

2.I want to figure out how these flags work and if this is the only file
where you can switch between miniAOD or AOD:
https://github.com/larsonka/SUSYBSMAnalysis-Zprime2muAnalysis/blob/MiniAOD
_74X/python/Zprime2muAnalysis_cfg.py
for sure, as suggested by Benjamin, it's better to have the AOD as default

Did you, Kurt, try to compile your code and produce a small pattuple? We
should check that it works. I don't think it will work because of the
possible issue 1. in CMSSW_4_12. But I could be wrong.

We should try to manage these checks before merging the PR.

ciao,
Raffaella

2015-11-17 23:47 GMT+01:00 Benjamin Radburn-Smith <notifications@github.com>
:

> In python/goodData_cff.py
> <https://github.com/cms-analysis/SUSYBSMAnalysis-Zprime2muAnalysis/pull/14#discussion_r45134444>
> :
>
> > @@ -10,7 +10,7 @@
> >                                    L1GtReadoutRecordTag = cms.InputTag('gtDigis')
> >                                    )
> >
> > -primaryVertexFilter = cms.EDFilter('GoodVertexFilter',
> > +primaryVertex = cms.EDFilter('GoodVertexFilter',
>
> Why the change of name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-analysis/SUSYBSMAnalysis-Zprime2muAnalysis/pull/14/files#r45134444>
> .
>